### DISCUSSION TITLE
Rebuild the database

### DISCUSSION THREAD

**Discussion by maxrodro** [Mimetis/Dotmim.Sync#961] [Q&A]
Hello @Mimetis , 

There is some command in Dotmim to rebuild the entire database, currently what we do is delete the database file and synchronize again, but I think it is bad practice.

Our version is 0.9.4 and we are using SQLite in our Android app

**Comment by Mimetis** [comment]
That's fine to delete the database if you are talking about "rebuild" the SQLite database.
In version `0.9.8`, we have the `DropaAllAsync` method that just remove everything from the database, except the tables.
